src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -16,44 +16,106 @@ UI Options does three things:
 
-1. [Download the UI Options library](https://github.com/fluid-project/infusion/releases/download/v2.0.0/infusion-uiOptions-2.0.0-source.zip)
-  (there is also a [minified version UI Options](https://github.com/fluid-project/infusion/releases/download/v2.0.0/infusion-uiOptions-2.0.0-minified.zip)
-   for deployment purposes).
+### Get Infusion UI Options Stable Release - Zip Package
+
+This method will download and install the most recent stable release of Infusion.
+
+1. Download the latest Infusion UI Options Zip package from the
+   [Infusion Releases page](https://github.com/fluid-project/infusion/releases). There are different packages of
+   Infusion - download the file "`infusion-uiOptions-X.X.X-source.zip`" if you are only interested in UI Options.
 2. Unzip the contents of the downloaded ZIP file to a location within your project. This will result in a new directory
    `infusion`.<div class="infusion-docs-note"><strong>Note:</strong> In this guide we will use the directory
    `my-project/lib/`.</div>
-3. Your `infusion` folder will include a single file containing all of the JavaScript you need
+3. Your `infusion` directory will include a single file containing all of the JavaScript you need
    (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later
    link to these files in your HTML files.
-4. Now that `infusion` is in your project directory, you can delete the `infusion-uiOptions-2.0.0.zip` (or similar name)
+4. Now that `infusion` is in your project directory, you can delete the `infusion-uiOptions-X.X.X.zip` (or similar name)
    from your download directory.
 
+### Get Infusion UI Options Stable Release with `npm`
+
+This method will download and install the most recent stable release of Infusion.
+
+1. From your project directory, run `npm i infusion`.
+2. Within the `node_modules/infusion/` directory in your project, run `npm install` to get Infusion's dependencies.
+3. Then build a version of Infusion with UI Options by running `grunt custom --include="uiOptions" --name="uiOptions"`.
+   This will create a build of Infusion in the `node_modules/infusion/build/` directory. Note:
+   [Get Grunt](https://gruntjs.com/) if you don't have it installed.
+4. Your `node_modules/infusion/build` folder will include a single file containing all of the JavaScript you need
+   (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later
+   link to these files in your HTML files.
+5. Rename the directory from `build` to `infusion`, and move (or copy) the `build` directory to your project (i.e.
+   `my-project/lib/infusion`).
+6. Optional: You can now safely delete the `infusion` directory created in step 1.
+
+#### Summary of commands:
+
+```bash
+npm i infusion
+cd node_modules/infusion/
+grunt custom --include="uiOptions" --name="uiOptions"
+mv build/ my-project/lib/infusion
+```
```

Comment:
I'm not sure this is correct. Maybe it's because we've moved the "latest" tag to point at a dev release.

Anyways, I think the NPM method should talk about installing via NPM, maybe assuming they are using a package.json file themselves. And it should not the parts that will need to be exposed or copied over from the "node modules" directory. For example sourcing the compiled JS file and Assets from "Dist" and any other material like templates and message bundles from source.